File: Ejercicios GO/12.Operaciones.go
package main

import "fmt"

func main() {
	var A, B float64 // Declaramos las variables A y B como float64 para permitir números decimales.

	// Solicitar el primer valor al usuario
	fmt.Println("Ingrese valor para A: ")
	fmt.Scanln(&A) // Usamos &A para pasar la dirección de la variable.

	// Solicitar el segundo valor al usuario
	fmt.Println("Ingrese valor para B: ")
	fmt.Scanln(&B) // Usamos &B para pasar la dirección de la variable.

	// Realizar operaciones
	suma := A + B
	fmt.Println("La suma de los números es:", suma)

	res := A - B
	fmt.Println("La resta de los números es:", res)

	multi := A * B
	fmt.Println("La multiplicación de los números es:", multi)

	if B != 0 { // Verificamos que B no sea cero para evitar división por cero.
		div := A / B
		fmt.Println("La división de los números es:", div)
	} else {
		fmt.Println("No se puede dividir por cero.")
	}

	igual := A == B
	fmt.Println("Los números son iguales:", igual)

	// Comparar A y B
	if A > B {
		fmt.Println("El número menor es:", B)
		fmt.Println("El número mayor es:", A)
	} else if A < B {
		fmt.Println("El número menor es:", A)
		fmt.Println("El número mayor es:", B)
	} else {
		fmt.Println("Ambos números son iguales.")
	}
}

File: Ejercicios JavaScript/12.Operaciones.js
//funcion que permite capturar entradas del usuario para ejecutar en node.js
const prompt = require('prompt-sync')();
//En este caso se le pedira al usuario el dato de entrada definidos en dos variables.
let A = Number(prompt('ingrese valor:'))//Number: define el valor de la variable como un numero.
let B = Number(prompt('ingrese valor:'))
//Definimos las operaciones a realizar.
let suma = A+B
console.log('la suma de los numeros es: ',suma)
let res = A-B
console.log('la resta de los numeros es: ',res)
let multi = A*B
console.log('la multiplicacion de los numeros es: ',multi)
let div = A/B 
console.log('la division de los numeros es: ',div)

//Definimos una variable que compare el igual y el mayor entre A y B.
let igual = A == B
console.log('el numero', A, 'es igual a',B,'?',igual)//Genera un valor booleano para comparar A y B.

let mayor = (A>B) ? A : B //Devuelve el valor numerico si el valor es verdadero.
let menor = (A<B) ? A : B //Devuelve el valor numerico si el valor es falso.
console.log('el numero menor es: ',menor)
console.log('el numero mayor es: ',mayor)
//Para ejecutar en terminar: node "C:\Sena\Analisis y desarrollo de software\Ficha 3066478 segundo trimestre\Ejercicios JavaScript\12.Operaciones.js"